Connected Services/HarnessOpenMetricsAPIService/HarnessOpenMetricsAPI.cs

```diff
@@ -14,19 +14,21 @@
 namespace io.harness.cfsdk.HarnessOpenMetricsAPIService
 {
     using System = global::System;
-    using Serilog;
+    using Microsoft.Extensions.Logging;
```

Comment:
I'm assuming your team manually modified this autogenerated file to insert logging.